cmd/auth/login.go

```diff
@@ -136,6 +136,14 @@ depends on the existing profiles you have set in your configuration file
 			return err
 		}
 
+		// Bug fix: Preserve cluster_id from existing profile during login
+		// Prior to v0.256.0, the login command would overwrite the entire profile configuration,
+		// causing loss of cluster_id and other settings. Now we first retrieve the existing
+		// cluster_id before saving the new configuration to ensure it's preserved.
+		cfg.ClusterID, err = getClusterID(ctx, profileName, defaultConfigPath)
```

Comment:
There's a bug here. We cannot use `defaultConfigPath` here. We need to account for two other cases as well:
1. When the file does not exist.
2. When a user configures the filepath using `DATABRICKS_CONFIG_FILE`

Both will stop working after this change. You can look at the `loadOrCreateConfigFile` function for how this works today.